### DISCUSSION TITLE
Expose `minimumFractionDigits` for number masking

### DISCUSSION THREAD

**Discussion by Roxanoel** [beholdr/maska#234] [Ideas]
At the moment, for numbers, we can specify a locale, a maximum number of decimal places, and whether to include the sign. Under the hood, the `createFormatter` function does take in a param that gets passed to `Intl.NumberFormat` as the `minimumFractionDigits`, but it gets programmatically set and not exposed as a field for the number option. 
The reason why I noticed this is that I currently have a use case where, independent of locale (and not for a money field), I want the numbers in the field to always have exactly two decimal places, for example `5` should be `5.00`, `5.254` should be `5.25`, and so on. If the field was exposed, this would be much easier to achieve.

**Comment by beholdr** [comment]
I could expose new setting for `minimumFractionDigits`. But in this case (when `minimumFractionDigits` is equal to `maximumFractionDigits`) you will not be able to delete last digits, like: `5.50|` + press `backspace` = `5.5|0`, press `backspace` again = `5.|00`. Is this desirable behavior?
